clang/test/SemaHLSL/resource_binding_attr_error.hlsl

```diff
@@ -45,7 +45,7 @@ void foo2() {
   extern RWBuffer<float> U2 : register(u5);
 }
 // FIXME: expect-error once fix https://github.com/llvm/llvm-project/issues/57886.
-float b : register(u0, space1);
+// float b : register(u0, space1) {}
```

Comment:
Shouldn't we be adding the `// expected-error:` here rather than commenting this out?